src/lib/Scenes/Collection/Collection.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Sans } from "@artsy/palette"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface CollectionProps {
+  collection: any
```

Comment:
how come this is marked as "any" - isn't a "collection" a graphQL query here?

Reply:
Yup that's correct, good catch. I just left it as `any` as a placeholder while the relay compiler was catching up with the module renaming and quick directory changes.